source/message37-condensed.txt

```diff
@@ -1 +0,0 @@
-TODO: fix decoded text?
```

Comment:
Can you add some prose here? This file gets included into `message37.rst` in the _Interpretation_ section.